slash/core/fixtures/fixture_store.py

```diff
@@ -249,7 +253,7 @@ def get_fixture_by_argument(self, arg):
     def get_fixture_by_id(self, fixture_id):
         return self._fixtures_by_id[fixture_id]
 
-    def get_fixture_dict(self, fixture_names, namespace=None, get_values=True, skip_names=frozenset()):
+    def get_fixture_dict(self, fixture_names, namespace=None, get_values=True, skip_names=frozenset(), add_to_dict=True):
```

Comment:
Call this new parameter "add_as_keyword_param", since this is what it means?

Reply:
Perhaps the interface would be better as an extra argument of "extra_initialized_fixtures" or something...